src/App.vue

```diff
@@ -444,7 +449,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
 
   /** Fetches and stores the current JS date. */
   private async updateCurrentJsDate (): Promise<void> {
-    const jsDate = await this.getServerDate()
+    const jsDate = await DateUtilities.getServerDate()
```

Comment:
Moving date utilities from a Mixin to a Service.  Since these were written as pure functions they're easy to move around.